srcpkgs/purescript/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'purescript'
+pkgname=purescript
+version=0.14.0
+revision=1
+wrksrc="purescript-${version}"
+build_style=haskell-stack
+makedepends="ncurses-libtinfo-devel ncurses-devel zlib-devel"
+short_desc="Strongly typed language that compiles to javascript"
+maintainer="Wayne Van Son <waynevanson@gmail.com>"
+license="GPL-3.0-or-later"
```

Comment:
I took a look, and the PureScript compiler is licensed under the 3-clause BSD license, not GPLv3 or later.